velox/exec/tests/AssertQueryBuilderTest.cpp

```diff
@@ -125,4 +126,31 @@ TEST_F(AssertQueryBuilderTest, hiveSplits) {
       .split(buildScanId, makeHiveConnectorSplit(buildFile->path))
       .assertResults("SELECT 2");
 }
+
+TEST_F(AssertQueryBuilderTest, encodedResults) {
+  VectorFuzzer::Options opts;
+  opts.vectorSize = 1000;
+  opts.nullRatio = 0.1;
+
+  VectorFuzzer fuzzer(opts, pool_.get());
+
+  // Dict(Array).
```

Comment:
Also is it possible to create one for Constant?

Reply:
Sure.